science/libfec/Portfile

```diff
@@ -12,22 +12,16 @@ long_description    implements several popular forward error correction \
     (FEC) algorithms and several low-level routines useful in modems \
     implemented with digital signal processing (DSP).
 
-github.setup fblomqvi libfec 9de5ef59bcdb6d97805cc93a674a6778c3392cfb
+github.setup fblomqvi libfec 076324752f842191664b05ef82a8be7dee6ecb0d
 version   20191015-[string range ${github.version} 0 7]
```

Comment:
The date in the `version` needs to be changed to 20200118.

Reply:
ops. thank you @ryandesign